node/core/approval-voting/src/criteria.rs

```diff
@@ -44,6 +44,8 @@ pub struct OurAssignment {
 	validator_index: ValidatorIndex,
 	// Whether the assignment has been triggered already.
 	triggered: bool,
+	// Claimed core indices.
+	pub claimed_core_indices: Vec<CoreIndex>,
```

Comment:
better practice to add a getter; mixing public and private fields always looks weird to me.

Can this legally be empty?

Reply:
That's a good point. No it cannot be empty.